Sourcery/Models/Type.swift

```diff
@@ -24,23 +24,25 @@ class Type: NSObject {
     /// Name in parent scope
     var localName: String
 
-    /// All type static variables
-    var staticVariables: [Variable]
-
     /// All instance variables
     var variables: [Variable]
 
     /// Annotations, that were created with // sourcery: annotation1, other = "annotation value", alterantive = 2
     var annotations: [String: NSObject] = [:]
 
+    /// All type static variables
+    var staticVariables: [Variable] {
+        return variables.filter({ $0.isStatic })
+    }
+
     /// Only computed instance variables
     var computedVariables: [Variable] {
-        return variables.filter { $0.isComputed }
+        return variables.filter { $0.isComputed && !$0.isStatic }
```

Comment:
Something not related to this pr. I was wondering why do we need a stored `isComputed` flag? Can't we just check for write access to be not `none`? It looks like if the property is not computed SourceKitten always returns setter access level, it does not return it only for computed properties.